### DISCUSSION TITLE
Vulnerability question

### DISCUSSION THREAD

**Discussion by imaggyy** [vercel/next.js#57356] [Help]
### Summary

I'm running an app with Next.js 7.0.3

I get a vulnerability warning for [CVE-2021-43803](https://github.com/advisories/GHSA-25mp-g6fv-mqxx) which has versions 0.9.9 onwards listed as affected.

However, the [v12.0.5 release notes](https://github.com/vercel/next.js/releases/v12.0.5) state it's only Next.js versions above v11.1.0 and below v12.0.5 that are affected.

Is 7.0.3 affected by this?

Thanks everyone

**Comment by icyJoseph** [comment]
If I remember correctly, that vulnerability was because starting with Node 15, unhandled rejections crash the process. 

I guess the issue is really exposed when you have a Next.js version that doesn't handle rejections correctly, and that allows them to happen in the first place, and at the same time you have Node 15+.

I guess, if you are not using Node 15+, you might be fine... you'll see warnings about unhandled rejections, but the server won't crash.

According to this test, from the commits that fixed this issue, you could try to see if the `*` path is available:

- https://github.com/vercel/next.js/blob/6d98b4fb4315dec1badecf0e9bdc212a4272b264/test/integration/production/test/security.js#L29-L55

It'd be nice to know of a payload that did crash the server though :/ to double check that there's no vulnerability.

I'd say, avoid Node 15+ if possible, just to prevent server crash attacks.
